core/templates/pages/library-page/library-page.component.html

```diff
@@ -58,7 +58,7 @@ <h1 class="oppia-classroom-h1" translate="I18N_CLASSROOM_CALLOUT_HEADING_1"></h1
             <p class="oppia-classroom-p" translate="I18N_CLASSROOM_CALLOUT_PARAGRAPH_1"></p>
             <a class="btn oppia-dashboard-intro-button oppia-transition-200" translate="I18N_CLASSROOM_CALLOUT_BUTTON" ng-href="/learn/math" href="/learn/math"></a>
           </div>
-          <picture class="d-flex">
+          <picture class="classroom-div-image">
```

Comment:
Let's call this the `classroom-promo-image`

Reply:
done